presto-parser/src/main/java/com/facebook/presto/sql/tree/Insert.java

```diff
@@ -26,18 +26,25 @@
     private final QualifiedName target;
     private final Query query;
     private final Optional<List<String>> columns;
+    private final boolean triggeredByRefresh;
```

Comment:
This is not part of the grammar and thus should not go into the AST. The AST should be a representation of the grammar, nothing more.